.github/PULL_REQUEST_TEMPLATE

```diff
@@ -0,0 +1,18 @@
+### :tophat: What is the goal?
```

Comment:
This way every time we create a new pull request we'll use this beautiful template :)